translator/tocwconfig/sampleConfig/appsignals_and_eks_config.yaml

```diff
@@ -503,6 +503,7 @@ processors:
         k8snode:
             auth_type: serviceAccount
             context: ""
+            kube_config_path: ""
```

Comment:
nit: Could we set `omitempty` for these fields? If we have control over it, I'd rather we didn't increase the size of our YAML.